configuration/assembly.yaml

```diff
@@ -28,11 +28,13 @@ METASPADES_illumina_max_k: 99
 #     each successive attempt will increase per-sample memory by 6 Gibabytes
 #     until it succeeds or reaches max_attempts from snakemake.
 #     Please make sure that there is enough RAM on the server.
+# Custom k-list for assembly: k must be odd, in the range 15-255, increment <= 28, fx. 21,29,39,59,79,99,119,141
 MEGAHIT_memory: 10
 MEGAHIT_threads: 32
 MEGAHIT_presets:
  - meta-sensitive
  - meta-large
+MEGAHIT_custom_k_list:
```

Comment:
I was thinking of our discussion. Could it be an idea to offer the following format?
```
MEGAHIT_presets:
 - meta-sensitive
 - meta-large
 - meta-custom-1: 21,29,39,59,79,99
 - meta-custom-2: 21,29,39,59,79,99,121
 ```
Too much?

Reply:
Sure, I could extend it into allowing multiple custom settings. It could maybe even called meta-custom-<whatever>, no need to use ints. Just no underscores.

Reply:
Implemented in https://github.com/arumugamlab/MIntO/pull/12/commits/bc009d48e2b5e40a576ae16bd9acfbfc1d677071

Reply:
I like this!